### PR TITLE
Update net6-rc1 with latest main

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

File: xml/System.Security.Cryptography/DSASignatureFormat.xml [hunk 1]
```diff
@@ -49,6 +49,20 @@
           <para>The signature format from IEEE P1363, which produces a fixed size signature for a given key.</para>
           <para>This signature format encodes the <c>(r, s)</c> tuple as the concatenation of the big-endian representation of <c>r</c> and the big-endian representation of <c>s</c>. Each of these values is encoded using the number of bytes required to encode the maximum integer value in the key's mathematical field. For example, an ECDSA signature from the curve <c>secp521r1</c> (a 521-bit field) encodes each of <c>r</c> and <c>s</c> as 66 bytes, and produces a signature output of 132 bytes.</para>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/DSASignatureFormat.xml:50]:
```suggestion
```

File: xml/System.Security.Cryptography/DSASignatureFormat.xml [hunk 2]
```diff
@@ -73,6 +87,20 @@
           <para>The signature format from IETF RFC 3279, which produces a variably-sized signature.</para>
           <para>This signature format encodes the <c>(r, s)</c> tuple as the DER encoding of <c>SEQUENCE(INTEGER(r), INTEGER(s))</c>. Because the length of a DER INTEGER encoding varies according to the value being encoded, this signature format does not produce a consistent signature length. Signatures in this format always start with <c>0x30</c>, and on average are 7 bytes longer than signatures in the <see cref="F:System.Security.Cryptography.DSASignatureFormat.IeeeP1363FixedFieldConcatenation" /> format.</para>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/DSASignatureFormat.xml:88]:
```suggestion
```

File: xml/System.Security.Cryptography/RSAEncryptionPaddingMode.xml [hunk 3]
```diff
@@ -85,6 +85,16 @@ The `Oaep` mode corresponds to the RSAES-OEAP encryption scheme described in the
           <para>Optimal Asymmetric Encryption Padding. It is recommended for new applications.</para>
           <para>This mode corresponds to the RSAES-OEAP encryption scheme described in the PKCS #1 RSA Encryption Standard. It is recommended for new applications.</para>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/RSAEncryptionPaddingMode.xml:86]:
```suggestion
```

File: xml/System.Security.Cryptography/RSAEncryptionPaddingMode.xml [hunk 4]
```diff
@@ -125,6 +135,16 @@ The `Oaep` mode corresponds to the RSAES-OEAP encryption scheme described in the
           <para>PKCS #1 v1.5. It is supported for compatibility with existing applications.</para>
           <para>This mode corresponds to the RSAES-PKCS1-v1_5 encryption scheme described in the PKCS #1 RSA Encryption Standard. It is supported for compatibility with existing applications.</para>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/RSAEncryptionPaddingMode.xml:136]:
```suggestion
```

File: xml/System.Security.Cryptography/RSASignaturePaddingMode.xml [hunk 5]
```diff
@@ -87,6 +87,16 @@
           <para>PKCS #1 v1.5.</para>
           <para>This corresponds to the RSASSA-PKCS1-v1.5 signature scheme of the PKCS #1 RSA Encryption Standard. It is supported for compatibility with existing applications.</para>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/RSASignaturePaddingMode.xml:88]:
```suggestion
```

**Comment by AaronRobinsonMSFT** [comment]
`System.Runtime.InteropServices` LGTM.

**Pull request merged**